wordpress-mobile/WordPress-iOS
# Move application(_:open:options:) to Swift

Refs #11152

### To test:
 - visit [this test site](http://heagy.com/urltest) (in simulator or a device with this build)
 - tap the links to test viewing stats or a specific post respectively to ensure those existing functions continue to work
 - login via Google to ensure _that_ still works
 - to test debug crashing you'd need to dig out our Crashlytics app key. Let me know if you want to do that 😜 

### Update release notes:

- [x] There are no user facing changes
